### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by dimqua**

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="error_media_upload_size">Файл должен быть не больше 4 Мбайт.</string>
+    <string name="error_media_upload_type">Данный тип файла не может быть загружен.</string>
+    <string name="error_media_upload_opening">Файл не может быть открыт.</string>
+    <string name="error_media_upload_permission">Необходимо разрешение на чтение медиаконтента.</string>
+    <string name="error_media_download_permission">Необходимо разрешение для хранения медиаконтента.</string>
+    <string name="error_media_upload_image_or_video">Изображения и видео не могут быть прикрекплены к статусу одновременно.</string>
+    <string name="error_media_upload_sending">Загрузка не удалась.</string>
+    <string name="error_report_too_few_statuses">Должен быть выбран хотя бы один статус.</string>
+
+    <string name="title_home">Дом</string>
```
Comment (charlag) [inline app/src/main/res/values-ru/strings.xml]: "Дом" sounds weird here, isn't it?

File: app/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="title_notifications">Уведомления</string>
+    <string name="title_public_local">Локальная лента</string>
+    <string name="title_public_federated">Глобальная лента</string>
+    <string name="title_thread">Тред</string>
+    <string name="title_tag">#%s</string>
+    <string name="title_statuses">Посты</string>
+    <string name="title_follows">Подписки</string>
+    <string name="title_followers">Подписчики</string>
+    <string name="title_favourites">Понравившееся</string>
+    <string name="title_mutes">Список глушения</string>
```
Comment (charlag) [inline app/src/main/res/values-ru/strings.xml]: Sounds pretty weird as well

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (dimqua) [inline app/src/main/res/values-ru/strings.xml]: @charlag yes, it is, but are there better options?

File: app/src/main/res/values-ru/strings.xml (hunk 2 above)
Comment (dimqua) [inline app/src/main/res/values-ru/strings.xml]: @charlag I took this from Russian translation of Mastodon (for compatibility reasons).

**Comment by Vavassor** [comment]
Thanks so much for contributing this, @dimqua!

**Pull request merged**